star-rx72n-firmware/lib/rx_frame/src/rx_frame.c

```diff
@@ -0,0 +1,284 @@
+/**
+ * @file rx_frame.c
```

Comment:
I dont like how this has magic number that are not documented, `8`, `24`, etc seem magic same with `0xFF` we need a comment or this should be in an enum or a macro if needed so we can know why we are shifting like this